eZ/Bundle/EzPublishCoreBundle/Features/QueryController/query_controller.feature

```diff
@@ -0,0 +1,55 @@
+Feature: Query controller
+    In order to simplify listing items from the repository
+    As a developer
+    I want to run repository queries from content views
+
+Scenario: A content view can be configured to run and render a query
+    Given the following configuration:
+    """
+    ezpublish:
+        default:
+            location_view:
+                full:
+                    folder:
+                        match:
+                            Identifier\ContentType: 'folder'
+                        controller: ez_query:locationQueryAction
+                        template: '@EzPlatformBehatBundle/Resources/views/query_controller.html.twig'
+                        params:
+                            query: 'LocationChildren'
+                            queryParameters:
+                                parentLocationId: '@=location.id'
+                            variable: 'children'
```

Comment:
maybe results_variable will be more clear at first. with only variable you may think this is a variable for the query (specially if you haven't read the doc, of course) ;)